caramel/config.py

```diff
@@ -0,0 +1,26 @@
+#! /usr/bin/env python
+# vim: expandtab shiftwidth=4 softtabstop=4 tabstop=17 filetype=python :
+
+import os
+
+
+def add_inifile_argument(parser):
+    default_ini = os.environ.get("CARAMEL_INI")
+
+    parser.add_argument(
+        nargs="?",
+        help="Path to a specific .ini-file to use as config",
+        dest="inifile",
+        default=default_ini,
+        type=str,
+    )
+
+
+def check_inifile_path_set(args):
```

Comment:
Check if this could be turned into an argparse action instead, last part of "action" documentation.

1: https://docs.python.org/3/library/argparse.html#action

Reply:
It could indeed and it feels like a better solution.